foodnet/membership/admin.py

```diff
@@ -1,19 +1,15 @@
 from django.contrib import admin
-from django.contrib.auth.admin import UserAdmin
 
 
-from .models import User, UserProfile, Department, DepartmentMembership,\
-    Invitation, Member, MemberCategory
-
-
-class UserProfileInline(admin.StackedInline):
-    model = UserProfile
-    can_delete = False
-    exclude = ['member', ]
-
-
-class UserAdmin(UserAdmin):
-    inlines = (UserProfileInline, )
+from .models import (
+    UserProfile,
+    Department,
+    DepartmentMembership,
+    DepartmentInvitation,
+    Account,
+    AccountCategory,
+    AccountMembership,
+)
```

Comment:
As far as I remember there is no python proposal which would enforce one style over the other so please respect others formating preferences too...